app/views/ladder/utils.coffee

```diff
@@ -3,6 +3,8 @@
 module.exports.teamDataFromLevel = (level) ->
   alliedSystem = _.find level.get('systems', true), (value) -> value.config?.teams?
   teamNames = (teamName for teamName, teamConfig of alliedSystem.config.teams when teamConfig.playable)
+  if teamNames[0] is 'ogres' and teamNames[1] is 'humans'
```

Comment:
Could `teamNames` ever be undefined and lead to a "Cannot read property" error?

Reply:
Nah, this list comprehension would lead to an empty list.